packages/core/.storybook/preview.js

```diff
@@ -16,5 +18,8 @@ export const parameters = {
   previewTabs: {
       canvas: { hidden: true }
   },
-  viewMode: 'docs'
+  viewMode: 'docs',
+	docs: {
+    theme: mayflowerTheme,
```

Comment:
nice catch! 

Interesting, I was referencing their config doc and it says to import in the manager.js instead of preview.js
https://storybook.js.org/docs/react/configure/theming

Where did you find this?

Reply:
Here: https://storybook.js.org/docs/react/configure/theming#theming-docs

Reply:
huh thanks! I was looking at the global config part